### PR TITLE
Adds new target for Happy Model ES915TX with power adjustments

### PR DESCRIPTION
**Pull request by wvarty**

### CODE REVIEW

File: src/lib/POWERMGNT/POWERMGNT.cpp [hunk 1]
```diff
@@ -7,8 +7,8 @@ extern SX127xDriver Radio;
 extern SX1280Driver Radio;
 #endif
 
-#ifdef TARGET_R9M_TX
-extern R9DAC R9DAC;
+#if defined(TARGET_R9M_TX) || defined(TARGET_TX_ES915TX)
+extern DAC DAC;
```
Comment (cruwaller) [inline src/lib/POWERMGNT/POWERMGNT.cpp:11]: Change variable name different than class name. This is not good habbit!

File: src/src/tx_main.cpp [hunk 2]
```diff
@@ -35,9 +35,9 @@ SX1280Driver Radio;
 #include "ESP32_WebUpdate.h"
 #endif
 
-#ifdef TARGET_R9M_TX
+#if defined(TARGET_R9M_TX) || defined(TARGET_TX_ES915TX)
 #include "DAC.h"
-R9DAC R9DAC;
+DAC DAC;
```
Comment (cruwaller) [inline src/src/tx_main.cpp:40]: Variable name

File: src/lib/DAC/DAC.cpp [hunk 3]
```diff
@@ -15,76 +19,79 @@ int R9DAC::LUT[8][4] = {
...
+#endif
 
-uint32_t R9DAC::CurrVoltageMV = 0;
-uint8_t R9DAC::CurrVoltageRegVal = 0;
-uint8_t R9DAC::SDA = 0;
-uint8_t R9DAC::SCL = 0;
-uint8_t R9DAC::ADDR = 0;
-
-DAC_STATE_ R9DAC::DAC_STATE = UNKNOWN;
+#if defined(TARGET_TX_ES915TX)
```
Comment (cruwaller) [inline src/lib/DAC/DAC.cpp:24]: `#elif`

File: src/lib/DAC/DAC.cpp [hunk 4]
```diff
@@ -1,10 +1,14 @@
 
-#ifdef TARGET_R9M_TX
+#if defined(TARGET_R9M_TX) || defined(TARGET_TX_ES915TX)
```
Comment (cruwaller) [inline src/lib/DAC/DAC.cpp:2]: Would it better to define
`GPIO_PIN_DAC_SCL` and `GPIO_PIN_DAC_SDA` 
Then use those here to make this a bit generic? Maybe move those arrays into target.h (define)?

File: src/lib/POWERMGNT/POWERMGNT.cpp (hunk 1 above)
Comment (wvarty) [reply src/lib/POWERMGNT/POWERMGNT.cpp:11]: > Change variable name different than class name. This is not good habbit!

I agree. This was a batch "find and replace", so I didn't pick it up.

File: src/lib/DAC/DAC.cpp (hunk 4 above)
Comment (wvarty) [reply src/lib/DAC/DAC.cpp:2]: Probably, but that's more work than I'm willing to do in this PR...

Comment (cruwaller) [reply src/lib/DAC/DAC.cpp:2]: ok, let's do it later.

**Review by cruwaller — APPROVED** [review]

File: src/lib/POWERMGNT/POWERMGNT.cpp (hunk 1 above)
Comment (AlessandroAU) [reply src/lib/POWERMGNT/POWERMGNT.cpp]: my fault

**Pull request merged**